news/103.new.rst

```diff
@@ -0,0 +1,3 @@
+Add hook for ``plotly`` to collect data files and hidden `pandas`, `cmath`, and `plotly.validator` imports
+Add hooks for ``dash`` and related packages to collect data files and hook for meta-data from ``flask-compress``
+Add hook for ``dash_bootstrap_components`` to collect data files
```

Comment:
RestructuredText ignores single line breaks so these sentences will all render together into one long sentence. You'll need to add an extra blank line between each one.
```suggestion
Add hook for ``plotly`` to collect data files and hidden `pandas`, `cmath`, and `plotly.validator` imports

Add hooks for ``dash`` and related packages to collect data files and hook for meta-data from ``flask-compress``

Add hook for ``dash_bootstrap_components`` to collect data files
```